.golangci.yml

```diff
@@ -18,6 +22,22 @@ linters-settings:
       - prefix(github.com/aws/eks-anywhere)
     custom-order: true
     skip-generated: false
+  godot:
+    exclude:
+      # Exclude todo comments.
+      - "^todo:"
```

Comment:
Is this regex? Wanna make it case insensitive?

Reply:
Good idea, done